manifold_airflow_dags/tupress_ingest_dag.py

```diff
@@ -105,10 +105,25 @@ def calculate_file_to_move(**context):
 #
 REMOVE_CACHED_DELTAS= BashOperator(
     task_id='remove_cached_deltas',
-    bash_command="rm /tmp/%s" % "{{ ti.xcom_pull(task_ids='get_file_to_transfer').replace(' ', '\ ') }}",
+    bash_command="rm { TUPRESS_WEB_PATH }/%s" % "{{ ti.xcom_pull(task_ids='get_file_to_transfer').replace(' ', '\ ') }}",
     dag=DAG
 )
 
+#
+# Sync TUPress Blog
+#
+sync_blog = f"""
+sudo su - { TUPRESS_USER_NAME } bash -c \
+ "cd /var/www/tupress &&\
+ RAILS_ENV=production bundle exec rake sync:feeds
+"""
```

Comment:
I think it will be easier to maintain this in the future if we just move this code to a proper bash script like we do in cob_datapipeline and funnel_cake.

Reply:
I agree, but this is a stopgap fix. We will implement a better solution as we develop the new version of TUPress website.